nipoppy/trackers/run_tracker.py

```diff
@@ -103,15 +119,19 @@ def run(global_config_file, dash_schema_file, pipelines, run_id=1):
     parser.add_argument('--global_config', type=str, help='path to global config file for your nipoppy dataset', required=True)
     parser.add_argument('--dash_schema', type=str, help='path to dashboard schema to display tracker status', required=True)
     parser.add_argument('--pipelines', nargs='+', help='list of pipelines to track', required=True)
+    parser.add_argument('--session_id', type=str, help='session_id', required=True)
```

Comment:
If we are not looping over sessions, then the `bagel.csv` will only have one session. I'm not sure that's the best way to go since we will have to combine bagels together for the dashboard

Reply:
Oh yes, that's true. I meant add a default option for `--session_id = ALL`. I think it's good to have option to select only one session for testing and also until we have standardized `session` vs `session_id` variables.

Reply:
Ok that makes sense. So we need to restore the loop over `session_ids`

Reply:
Yup, fixed!